exercises/isogram/isogram_test.rb

```diff
@@ -77,6 +77,6 @@ def test_duplicated_character_in_the_middle
 
   def test_bookkeeping
     skip
-    assert_equal 3, BookKeeping::VERSION
+    assert_equal 4, BookKeeping::VERSION
```

Comment:
If messages are the only thing being changed, and not the tests themselves, then VERSION should not change.